repositories/vaccineOps.js

```diff
@@ -0,0 +1,20 @@
+const axios = require("axios");
+
+module.exports = {
+  getUsData: async (state) => {
+    //Get current data for most recent date
+    var yesterday = new Date(new Date().valueOf() - 1000 * 60 * 60 * 24)
```

Comment:
we should avoid using `var` (use `let` instead if we need to reassign the variable), and we should prefer using `const` if the variable isn't being reassigned
```suggestion
    const yesterday = new Date(new Date().valueOf() - 1000 * 60 * 60 * 24)
```